js/client-chat.js

```diff
@@ -881,13 +881,8 @@
 
 			case 'avatar':
 				var parts = target.split(',');
-				var avatarString = toId(parts[0]);
-				var avatar = parseInt(avatarString, 10);
-				if (avatar) {
-					Dex.prefs('avatar', avatar);
-				} else if (['bw2elesa', 'teamrocket', 'yellow', 'zinnia', 'clemont'].indexOf(avatarString) > -1) { // custom avatar exceptions
-					Dex.prefs('avatar', '#' + avatarString);
-				}
+				var avatar = toId(parts[0]);
```

Comment:
Close, but avatars can have `-` in them.

Reply:
The server is currently using a hardcoded regex, you can just copy/paste it.

Reply:
Whoops, slipped my mind when doing this.